core/src/validator.rs

```diff
@@ -1680,29 +1680,28 @@ fn active_vote_account_exists_in_bank(bank: &Bank, vote_account: &Pubkey) -> boo
 fn check_poh_speed(
     genesis_config: &GenesisConfig,
     maybe_hash_samples: Option<u64>,
-) -> Result<(), String> {
+) -> Result<(), ValidatorError> {
     if let Some(hashes_per_tick) = genesis_config.hashes_per_tick() {
         let ticks_per_slot = genesis_config.ticks_per_slot();
```

Comment:
I'm assuming this is what we'll clean up later to pull from bank

Reply:
Yep, we'll grab hashes per tick and ticks per slot from Bank instead of genesis.